content/departments/product-engineering/engineering/cloud/iam-and-admin-exp/index.md

```diff
@@ -47,24 +51,16 @@ Both technical and presentational sides of pricing. This includes both the prese
 
 The end-to-end self-service process on paying for Sourcegraph subscription on the cloud.
 
-#### Identity and access management
-
-We own both authentication and authorization to Sourcegraph.com, including both login and sign-up flow and user management on the cloud.
-
-Code-level authorization is enforced based on the repository permissions on the code host level and the Repository Management team owns this area. It is coupled with administration experience and team management, which created close collaboration between both teams.
-
 #### Usage reporting and entitlement enforcement
 
 Customer-facing and internal tools and features that provide transparency to how customers consume entitlements within their current subscription, including entitlements limiting enforcing capabilities.
 
 ### Engineering ownership
 
-The list below contains features and services that already exist, and the Cloud SaaS team is responsible for keeping the light on them.
+The list below contains features and services that already exist, and the IAM & Admin Exp team is responsible for keeping the light on them.
```

Comment:
My understanding - correct me if I am mistaken, please - is that all features/svcs that the IAM & Admin Exp team is owning already exist, so my idea was to update this block just to have a quick list of what the team owns (based on the engineering_ownership.yaml file bellow)

Reply:
Should we just link to the engineering_ownership page? To have a single source of truth, also we've stated the high-level focus (IAM and Admin Exp).

Reply:
I like your suggestion, @unknwon. Having a single source of truth is better.